frontends/infinite-corridor/src/api/learning-resources/hooks.ts

```diff
@@ -494,28 +481,6 @@ const useMoveListItem = (mode: "userlist" | "stafflist") => {
       keys.staffList.id(id).itemsListing.all
   return useMutation({
     mutationFn,
-    onMutate: vars => {
```

Comment:
This `onMutate` was performing an optimistic update to the item order prior to API request finishing. See `ItemsListing.tsx` for more about this change.